File: hacker_rank/algorithms/intro.cs
using System;
using System.Collections.Generic;
using System.IO;
class Solution {
    static void Main(String[] args) {
        string s = System.Console.In.ReadLine();
        string n = System.Console.In.ReadLine();
        string strings = System.Console.In.ReadLine();
        string[] nums = strings.Split(' ');
        for(var i = 0; i < nums.Length; i++) {
            if(nums[i] == s) {
               System.Console.WriteLine(i);
                return;
            }
        }
        /* Enter your code here. Read input from STDIN. Print output to STDOUT. Your class should be named Solution */
    }
}

File: hacker_rank/algorithms/intro.js
function processData(input) {
  var lines = input.split('\n');
  var arr = lines[2].split(' ');
  for (var i = 0; i < arr.length; i++) {
    if (arr[i] === lines[0]) {
      console.log(i);
      return;
    }
  }
  console.log(arr);
  //Enter your code here
}
